src/test/javascript/spec/component/about-us/about-us.component.spec.ts

```diff
@@ -44,7 +44,7 @@ describe('AboutUsComponent', () => {
 
         const getStaticJsonFromArtemisServerStub = sandbox.stub(staticContentService, 'getStaticJsonFromArtemisServer').returns(of(new AboutUsModel([], [])));
         const getProfileInfoSub = sandbox.stub(profileService, 'getProfileInfo');
-        getProfileInfoSub.returns(new BehaviorSubject<ProfileInfo | undefined>({ inProduction: false, sshCloneURLTemplate: 'ssh://git@testserver.com:1234/' } as ProfileInfo));
+        getProfileInfoSub.returns(new Observable<ProfileInfo>());
```

Comment:
```suggestion
        getProfileInfoSub.returns(new BehaviorSubject<ProfileInfo>({ inProduction: false, sshCloneURLTemplate: 'ssh://git@testserver.com:1234/' } as ProfileInfo).asObservable());
```

This seems to change the existing logic. Before it was returning a `BehaviorSubject` with a predefined object and now it returns an empty `Observable` stream.

Can you try to invoke `.asObservable()` on this `BehaviorSubject` and should return `Observable<T>`
https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/asobservable.md#rxobservableasobservable-%E2%93%A2

I would assume that this should help and make the tests pass.
I guess this change has to be applied on every breaking test.

Reply:
Yes, indeed this seems to do the trick.
(This is more intuitive and handy than trying to insert a starting value in an Observable with `startWith()`)
Great, thanks for sharing this knowledge👍

Moreover, the error that I had with `course-update.component.spec.ts` has disappeared. 
So now everything should be working.